brave-jdbc/src/main/java/com/github/kristofa/brave/jdbc/BraveJdbcListener.java

```diff
@@ -0,0 +1,152 @@
+package com.github.kristofa.brave.jdbc;
+
+import com.github.kristofa.brave.ClientTracer;
+import com.p6spy.engine.common.PreparedStatementInformation;
+import com.p6spy.engine.common.StatementInformation;
+import com.p6spy.engine.event.JdbcEventListener;
+
+import zipkin.Constants;
+import zipkin.TraceKeys;
+
+import java.net.Inet4Address;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.nio.ByteBuffer;
+import java.sql.SQLException;
+
+public final class BraveJdbcListener extends JdbcEventListener {
```

Comment:
I think you can extend [`SimpleJdbcEventListener`](https://github.com/p6spy/p6spy/blob/master/src/main/java/com/p6spy/engine/event/SimpleJdbcEventListener.java) to reduce code duplication